.taskcat.yml

```diff
@@ -4,10 +4,7 @@ project:
   package_lambda: false
   regions:
   - us-east-1
-  - us-west-2
-  - eu-central-1
-  - ap-southeast-2
-  s3_bucket: ''
+  s3_bucket: 'quickstart-reference-dburkhead'
```

Comment:
Please replace this with `$[taskcat_autobucket]`; Our tests will run and attempt to upload to your bucket, and will fail. `$[taskcat_autobucket]` is a psuedo-parameter placeholder for an auto-generated bucket name.

Reply:
@andrew-glenn Added another commit to switch to $[taskcat_autobucket]. Let me know if you see any other changes required.